TEStribute/__init__.py

```diff
@@ -159,16 +159,19 @@ def rank_services(
     # tes_info_drs will have all the drs costs & info for each TES
     tes_info_drs = {}
     for uri, info in tes_info.items():
-        tes_info_drs[uri] = sum_costs(
+        # TO-DO : here the older values of each drs object's cost are overwitten
+        #  fix the updation, though the normal dicts are NOT overwritten
+        tes_info_drs.update({uri: sum_costs(
```

Comment:
here if you notice the return from the compute_costs function is overwritten somehow, but only for a part of the dictionary, I can't seem to figure out why for now

Reply:
I'll have a look at this after merging. The docs are fine now.